Adds doc page for AI Agents (#66)
getkoala/docs: product-manual/koala-agents.mdx

---
title: "AI Agents"

> description: "Build a custom AI agent to automate your sales workflows"

Reviewer: ```suggestion
description: "Learn how to use AI Agents in Koala to automatically research and qualify leads and accounts."
```

---

> ## What is an agent?

Reviewer: ```suggestion
## What is an AI Agent?
```

> Agents are a way to automate your sales workflows. You can use them to simply answer questions about a profile or account, or to qualify leads and accounts.

Reviewer: ```suggestion
Agents are a way to automate lead and account research for your sales team. You can use them to simply answer questions about people or companies, and can use them to automatically qualify leads and accounts based on their research.
```

There are a few ways of creating an agent:
- When setting up qualification in a play
- AI agents on settings
- Research agents

Let's dive into each of these.

> ## Qualification agents

Reviewer: It's important to highlight (and repeat for both Qualification Agents and Research Agents) that an agent should be designed to answer 1 specific thing. Dont try to overload the agent with a bunch of questions. You can always combine multiple qualification agents to qualify leads/accounts, and this will produce better results than trying to ask too many things at once.

We can include some best practice about how to write your prompt, about when to use which model (and when in doubt use GPT 4o), etc.

You can use a qualification agent to automatically qualify leads and accounts based on research. 

### Creating an agent

To create an agent you can use for qualification, select or create a new agent in the third step of the `Create New Play` flow:

![Qualification agent](/images/product-manual/koala-agents/qualification.png)

To create an AI agent, you'll need to:

1. **Agent goal** - Define the purpose of your agent (required)
2. **Question/Prompt** - Ask the specific question you want answered (required)
3. **Research type** - Choose between "Companies" or "People" (required)
4. **Research individual people** - Option for people-focused research
5. **Response Type** - Select how the AI should format its answer
6. **Advanced Options** - Additional configuration settings

![Qualification agent](/images/product-manual/koala-agents/creating-ai-agent.png)

> For qualification agents, the **Response Type** is particularly important. Based on those you'll set up criteria for qualifying leads and accounts.

Reviewer: This is probably worth using a Note callout component.

I'd also highly recommend using boolean, number or picklist response types because these will be easier to create qualification conditions with.

### Qualification criteria and response types

Take the following example:

![Qualification agent](/images/product-manual/koala-agents/criteria-set-up.png)

The agent named `Sales Contracts > $50k` has response type `Boolean`, which means that the agent will only return a `Yes` or `No` answer.

You can then set up a rule to qualify a lead or account based on that answer.

**Response types**

We currently support the following response types:

- **String**: A text response for open-ended answers
- **Number**: A numerical value without currency formatting
- **Currency**: A monetary amount with the appropriate currency symbol
- **Percent**: A percentage value
- **Boolean**: A Yes/No or True/False response
- **Date**: A calendar date
- **List**: Multiple items returned as a list
- **Picklist**: Selection from predefined options

Keep in mind that the response type will affect the criteria you can set up.

> ## Research agents

Reviewer: I'd probably flip the order and introduce research agents first, and then qualification agents (which are simply research agents that take action on your behalf)

> You can use AI agents to answer questions about a profile or account. Then you can show in a few places:

Reviewer: Use "people" instead of the word "profiles". Customers dont know what a "profile" is (that's just our internal name for it in the database)

- In the plays list
- In the composer
- On the profile/account pages

### Displaying research in the plays list

You can add a research agent to the plays list by clicking on `Add AI Agent` in the top right corner of the plays list.

![Research agent](/images/product-manual/koala-agents/research-agents-play-list.png)

> ## Using resarch in composer

Reviewer: ```suggestion
## Using AI Research Agents with Coach
```

> After adding research agents to the plays list, you can use them in Koala Coach. You can select the relevant research knowledge to write personalized emails, Linkedin messages, and call scripts.

Reviewer: ```suggestion
After adding AI Research Agents to a Play, you can use them in Koala Coach. You can select the relevant research knowledge to write personalized emails, LinkedIn messages, and call scripts.
```
